README.md

```diff
@@ -139,6 +139,22 @@ Configure the project paths and other variables inside the `wpgulp.config.js` fi
 
 ![wpgulp config](https://on.ahmda.ws/f2ca9bb4a740/c)
 
+#### Add-on Styles Option
```

Comment:
This is not the right place for content like this. Kindly, create a new heading after installation call it, FAQs, ask a question and then answer it. 👍